src/utils/sendStatusJSONFormatter.js

```diff
@@ -0,0 +1,9 @@
+import statuses from 'statuses';
+
+function formattedSendStatus(statusCode) {
+  return this.contentType('application/json')
```

Comment:
Adicionar um comentário para ficar claro do que o `this` se trata.

```suggestion
  // this === app.response
  return this.contentType('application/json')
```